### PR TITLE
remove ninefold

### PR DESCRIPTION
**Pull request by lanej**
The Ninefold platform has been sunset

image

References #3942

### CODE REVIEW

File: lib/fog/bin.rb [hunk 1]
```diff
@@ -73,7 +73,6 @@ def collections
 require 'fog/bin/linode'
 require 'fog/bin/local'
 require 'fog/bin/bare_metal_cloud'
-require 'fog/bin/ninefold'
```
Comment (plribeiro3000) [inline lib/fog/bin.rb]: Don't you have to remove this bin file as well?

Comment (lanej) [inline lib/fog/bin.rb]: I can't seem to find it.  Do you know where it is?

Comment (plribeiro3000) [inline lib/fog/bin.rb]: You can find it [here](https://github.com/fog/fog/blob/master/lib/fog/bin/ninefold.rb). =)

Comment (lanej) [inline lib/fog/bin.rb]: odd. github file search failed me.

**Comment by plribeiro3000** [comment]
Noooice. Thanks!

I'm working on the build issue right now. Will have something in a couple of hours i guess.

**Comment by geemus** [comment]
Thanks!

**Pull request merged**